<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordGenerator</class>
 <widget class="QMainWindow" name="PasswordGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PasswordGenerator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="QCB_smallLetters">
            <property name="text">
             <string>Small Letters</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="QCB_capitalLetters">
            <property name="text">
             <string>Capital Letters</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="QCB_numbers">
            <property name="text">
             <string>Numbers</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="QCB_specialCharacters">
            <property name="text">
             <string>Special characters</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="QL_minimum">
              <property name="text">
               <string>Minimum: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="QSB_minimum">
              <property name="minimum">
               <number>4</number>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="QL_maximum">
              <property name="text">
               <string>Maximum: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="QSB_maximum">
              <property name="minimum">
               <number>4</number>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="value">
               <number>127</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="QL_numberOfPasswords">
              <property name="text">
               <string>Number of Passwords:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="QSB_numberOfPasswords">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="QPB_generatePassword">
        <property name="text">
         <string>Generate Passwords</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPlainTextEdit" name="QPTE_listOfPasswords">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="QPB_clear">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
